Orbitersdk/samples/ProjectApollo/Build/VC2017/LCC.vcxproj

```diff
@@ -127,7 +127,7 @@
       <AssemblerListingLocation>.\Debug\LCC/</AssemblerListingLocation>
       <ObjectFileName>.\Debug\LCC/</ObjectFileName>
       <ProgramDataBaseFileName>.\Debug\LCC/</ProgramDataBaseFileName>
-      <AdditionalIncludeDirectories>../../src_rtccmfd;../../src_sys;../../../../include;../../../../../Sound/OrbiterSound_SDK/VESSELSOUND_SDK/ShuttlePB_project;</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>C:\github\NASSP\Orbitersdk\samples\ProjectApollo\src_aux;../../src_rtccmfd;../../src_sys;../../../../include;../../../../../Sound/OrbiterSound_SDK/VESSELSOUND_SDK/ShuttlePB_project</AdditionalIncludeDirectories>

```

Comment:
Idem

Reply:
Nice job Visual Studio, the only thing I did through the GUI and the default behavior is crap...
At least now I know, I will be more careful in the future...